openedx_external_enrollments/edxapp_wrapper/backends/course_settings_j_v1.py

```diff
@@ -0,0 +1,79 @@
+"""Backend for course settings module."""
+import logging
+import time
+
+from cms.djangoapps.models.settings.course_metadata import CourseMetadata
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from openedx_external_enrollments.models import OtherCourseSettings
+from xmodule.modulestore.django import modulestore
+
+LOG = logging.getLogger(__name__)
+
+
+def update_course_settings(*args, **kwargs):  # pylint: disable=unused-argument
+    """
+    Updates external_course_id and external_platform from a course overview.
+    This function is called from edx-platform once advance course settings is successfully saved.
+    """
+
+    try:
+        course_overview = CourseOverview.get_from_id(kwargs.get('course_key'))
```

Comment:
Unnecessary variable as it's only used once. @ivanvgh

Reply:
Removed.